bundles/plugins/org.bonitasoft.studio.properties/src-test/java/org/bonitasoft/studio/properties/sections/subprocess/CallActivityHelperTest.java

```diff
@@ -0,0 +1,321 @@
+/**
+ * Copyright (C) 2015 Bonitasoft S.A.
+ * Bonitasoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2.0 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.bonitasoft.studio.properties.sections.subprocess;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.doReturn;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.bonitasoft.studio.common.emf.tools.ExpressionHelper;
+import org.bonitasoft.studio.common.emf.tools.ModelHelper;
+import org.bonitasoft.studio.common.repository.RepositoryAccessor;
+import org.bonitasoft.studio.diagram.custom.repository.DiagramRepositoryStore;
+import org.bonitasoft.studio.model.expression.builders.ExpressionBuilder;
+import org.bonitasoft.studio.model.process.AbstractProcess;
+import org.bonitasoft.studio.model.process.CallActivity;
+import org.bonitasoft.studio.model.process.ContractInputType;
+import org.bonitasoft.studio.model.process.Data;
+import org.bonitasoft.studio.model.process.InputMapping;
+import org.bonitasoft.studio.model.process.InputMappingAssignationType;
+import org.bonitasoft.studio.model.process.OutputMapping;
+import org.bonitasoft.studio.model.process.Pool;
+import org.bonitasoft.studio.model.process.builders.CallActivityBuilder;
+import org.bonitasoft.studio.model.process.builders.ContractBuilder;
+import org.bonitasoft.studio.model.process.builders.ContractInputBuilder;
+import org.bonitasoft.studio.model.process.builders.DataBuilder;
+import org.bonitasoft.studio.model.process.builders.InputMappingBuilder;
+import org.bonitasoft.studio.model.process.builders.JavaObjectDataBuilder;
+import org.bonitasoft.studio.model.process.builders.OutputMappingBuilder;
+import org.bonitasoft.studio.model.process.builders.PoolBuilder;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CallActivityHelperTest {
+
+    @Mock
+    private RepositoryAccessor repositoryAccessor;
+    @InjectMocks
```

Comment:
because it is the way eclipse generates it, fo rme both are not really readable so I can change it